components/client/home/study-here.js

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import {
+  LinkCarousel,
+  LinkCarouselLinks,
+  LinkCarouselLink,
+  LinkCarouselContent,
+  LinkCarouselItem,
+} from "@uoguelph/react-components/link-carousel";
+import undergraduate from "@/img/leah-weller-environmental-engineering-u-of-g.jpg";
+import graduate from "@/img/caroline-pottruff-landscape-architecture-u-of-g.jpg";
+import international from "@/img/ryan-ahlers-theatre-studies-u-of-g.jpg";
+import lifelong from "@/img/kathryn-knowles-jenna-schamowski-environmental-sciences-u-of-g.jpg";
+import Image from "next/image";
+import { tv } from "tailwind-variants";
+
+export const StudyHere = () => {
+  const classes = tv({
+    slots: {
+      link: "text-xl",
+      item: "",
+      image: "h-96",
+      caption:
+        "absolute bottom-0 left-0 z-10 text-white bg-linear-to-t from-black to-transparent p-4 pt-6 text-lg w-full",
+    },
+  });
+
+  const { link, item, image, caption } = classes();
+
+  return (
+    <LinkCarousel direction="right" stack>
+      <LinkCarouselLinks>
```

Comment:
On focus, the image should change to the one associated with the affiliated link (it works on hover, but not on focus)

<img width="1538" alt="image" src="https://github.com/user-attachments/assets/b25c67a0-3262-45b8-9c98-58a4ceba191b" />